main.go

```diff
@@ -70,61 +70,76 @@ func main() {
 	}
 }
 
-func check(e error) {
-	if e != nil {
-		panic(e)
-	}
+func get_kd_files(index int) (string, string) {
+	var keyfile = db + "/" + strconv.Itoa(index) + ".key"
+	var datafile = db + "/" + strconv.Itoa(index) + ".data"
```

Comment:
also in other places
```suggestion
	var datafile = filepath.Join(db, fmt.Sprintf("%d.data"))
```